RecoMET/METProducers/python/hcalnoiseinfoproducer_cfi.py

```diff
@@ -82,7 +82,14 @@
     lRBXRecHitR45Cuts = cms.vdouble(0.0, 1.0, 0.0, -0.5,   # equivalent to "fraction > 0.5"
                                     0.0, 0.0, 1.0, -0.5),  # equivalent to "energy fraction > 0.5"
     tRBXRecHitR45Cuts = cms.vdouble(0.0, 1.0, 0.0, -0.2,   # equivalent to "fraction > 0.2"
-                                    0.0, 0.0, 1.0, -0.2)   # equivalent to "energy fraction > 0.2"
+                                    0.0, 0.0, 1.0, -0.2),   # equivalent to "energy fraction > 0.2"
+
+    LaserMonDetTypeList = cms.vint32(3,3,3,3,3,3,3,3),
+    LaserMonIPhiList = cms.vint32(23,17,11,5,29,35,41,47),
+    LaserMonIEtaList = cms.vint32(0,0,0,0,0,0,0,0),
+
+    #LaserMonitorTSStart = cms.int32( 4 ),
```

Comment:
Are these commented out parameters needed: if so, please add a comment explaining why you keep them commented out, otherwise just remove